src/actions/teams.ts

```diff
@@ -740,3 +740,23 @@ export function membersMinusGroupMembers(teamID: string, groupIDs: Array<string>
         ],
     });
 }
+
+export function getInProductNotices(teamId: string, client: string, clientVersion: string): ActionFunc {
+    return bindClientFunc({
+        clientFunc: Client4.getInProductNotices,
+        params: [
+            teamId,
+            client,
+            clientVersion,
+        ],
+    });
+}
+
+export function updateNoticeAsViewed(noticeIds: string[]): ActionFunc {
```

Comment:
better name would be updateNoticesAsViewed since we accept an array

Reply:
Good call, Changed it